vllm/v1/worker/gpu_model_runner.py

```diff
@@ -2255,7 +2266,7 @@ def capture_model(self) -> None:
         # Trigger CUDA graph capture for specific shapes.
         # Capture the large shapes first so that the smaller shapes
         # can reuse the memory pool allocated for the large shapes.
-        with graph_capture(device=self.device):
+        with self.suppress_gc_collect(), graph_capture(device=self.device):
```

Comment:
Should we also call `gc.collect()` once here before starting?

In fact, it may be best to do
```python
gc.collect()
gc.freeze()
```

which may additionally help to reduce the time spent in automatic gc's.

IMO it shouldn't be necessary to call it manually periodically.